docs/basics/intro_information.ipynb

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
<p>theres probably some more NWB links you could add in here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>